docs/.vitepress/components/Term.vue

```diff
@@ -0,0 +1,53 @@
+<!-- Taken and modified from https://gitlab.com/eeriksp/vuepress-plugin-glossary -->
+
+<template>
+  <a :title="definition" :class="{ 'term-not-found': termNotFound, term: true }">
+    {{ displayText }}
+  </a>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import { removeTermLink } from "../utils/term-tag";
+import { data } from '../data/glossary.data.ts'
+
+const props = defineProps<{
+  term: string;
+  show?: string;
+}>();
+
+const termNotFound = ref(false);
+
+const definition = computed(() => {
+  const termWithSpaces = replaceUnderscoresWithSpaces(props.term);
+  const termDefinition = data[termWithSpaces];
+
+  if (termDefinition) {
+    return removeTermLink(termDefinition);
+  }
+
+  termNotFound.value = true;
```

Comment:
`computed` props should not have side effects. We'll want an additional `termText` property than the `termNotFound` boolean is derived from.